src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -280,11 +299,28 @@
                                    :limit-fiat   fiat-limit
                                    :limit-crypto crypto-limit})}]
          [routes/view
-          {:amount       amount-text
-           :routes       best-routes
-           :token        token
-           :input-value  @input-value
-           :fetch-routes #(fetch-routes % (current-limit))}]
+          {:from-values-by-chain   from-values-by-chain
+           :to-values-by-chain     to-values-by-chain
+           :affordable-networks    affordable-networks
+           :routes                 best-routes
+           :token                  token
+           :input-value            @input-value
+           :fetch-routes           #(fetch-routes % (current-limit) 2000)
+           :disabled-from-networks disabled-from-chain-ids
+           :on-press-from-network  (fn [chain-id _]
+                                     (let [disabled-chain-ids (if (contains? (set
+                                                                              disabled-from-chain-ids)
+                                                                             chain-id)
+                                                                (vec (remove #(= % chain-id)
+                                                                             disabled-from-chain-ids))
+                                                                (conj disabled-from-chain-ids
+                                                                      chain-id))
+                                           re-enabling-chain? (< (count disabled-chain-ids)
+                                                                 (count disabled-from-chain-ids))]
+                                       (when (or re-enabling-chain?
+                                                 (> (count affordable-networks) 1))
```

Comment:
`(> (count affordable-networks) 1)` -> (pos? (count affordable-networks) )`

Reply:
`pos?` checks if value is greater than zero, but here we are checking if it is greater than 1 🤔

Reply:
Yep I'm wrong. I realised this some time after but couldn't remember where I left the comment :)